osquery/tables/networking/linux/routes.cpp

```diff
@@ -131,6 +132,18 @@ void genNetlinkRoutes(const struct nlmsghdr* netlink_msg, QueryData& results) {
     case RTA_PRIORITY:
       r["metric"] = INTEGER(*(int*)RTA_DATA(attr));
       break;
+    case RTA_METRICS:
+      struct rtattr* xattr = static_cast<struct rtattr*> RTA_DATA(attr);
+      uint32_t xattr_size = RTA_PAYLOAD(attr);
```

Comment:
`   unsigned int RTA_PAYLOAD( `
Let's just used auto to be safe. Maybe uint32_t and unsigned int can be different.